roles/provision-data-sync-apb/tasks/provision-sync-server.yml

```diff
@@ -38,39 +69,11 @@
         value: "{{ postgres_user}}"
       - name: POSTGRES_DATABASE
         value: "{{ postgres_database }}"
+      - name: GRAPHIQL_SUBS_ENDPOINT
```

Comment:
SUBS? Could it be `WEBSOCKET`? It could be used for more than subscriptions.

Reply:
@wtrocki really it's a specific piece of config, and now that https://github.com/aerogear/data-sync-server/pull/50 is merged, they need to match.